src/Commands/core/CacheCommands.php

```diff
@@ -58,6 +58,23 @@ public function get($cid, $bin = 'default', $options = ['format' => 'json'])
         return new PropertyList($result);
     }
 
+    /**
+     * Invalidate by cache tags.
+     *
+     * @command cache:tags
+     * @param string $tags A comma delimited list of cache tags to clear.
+     * @aliases ct
+     * @bootstrap full
+     * @usage drush cache:tag node-12,user-4
+     *   Purge content associated with two cache tags.
+     */
+    public function tags($tags)
+    {
+        $tags = StringUtils::csvToArray($tags);
+        Cache::invalidateTags($tags);
+        $this->logger()->success(dt("Tnvalidated tag(s): '!list'.", ['!list' => implode(' ', $tags)]));
```

Comment:
Are we using "T" instead of "I" in "Tnvalidated" for a reason?

Reply:
Fixed. Thanks.